Cannot run tests in Windows shell

Maybe it's because of the difference in operating systems, but the straight forward way of running the tests did not work. I forked & cloned the repo, then

```bash
> npm install
> npm install -g mocha #not sure if it's necessary, did it anyways
> npm test
```

Now I can see in the log, that the build completes fast & without errors, but the tests don't even start:

```
E:\core-decorators.js>npm test

> core-decorators@0.9.1 test E:\core-decorators.js
> npm run build && mocha --compilers js:babel/register --require babel/polyfill 'test/**/*.spec.js'


> core-decorators@0.9.1 build E:\core-decorators.js
> babel --out-dir lib src

src\autobind.js -> lib\autobind.js
src\core-decorators.js -> lib\core-decorators.js
src\debounce.js -> lib\debounce.js
src\decorate.js -> lib\decorate.js
src\deprecate.js -> lib\deprecate.js
src\enumerable.js -> lib\enumerable.js
src\memoize.js -> lib\memoize.js
src\mixin.js -> lib\mixin.js
src\nonconfigurable.js -> lib\nonconfigurable.js
src\nonenumerable.js -> lib\nonenumerable.js
src\override.js -> lib\override.js
src\private\utils.js -> lib\private\utils.js
src\readonly.js -> lib\readonly.js
src\suppress-warnings.js -> lib\suppress-warnings.js
src\throttle.js -> lib\throttle.js
E:\core-decorators.js\node_modules\mocha\lib\utils.js:626
        throw new Error("cannot resolve path (or pattern) '" + path + "'");
        ^

Error: cannot resolve path (or pattern) ''test/**/*.spec.js''
    at Object.lookupFiles (E:\core-decorators.js\node_modules\mocha\lib\utils.js:626:15)
    at E:\core-decorators.js\node_modules\mocha\bin\_mocha:316:30
    at Array.forEach (native)
    at Object.<anonymous> (E:\core-decorators.js\node_modules\mocha\bin\_mocha:315:6)
    at Module._compile (module.js:435:26)
    at Object.Module._extensions..js (module.js:442:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:311:12)
    at Function.Module.runMain (module.js:467:10)
    at startup (node.js:134:18)
    at node.js:961:3
npm ERR! Test failed.  See above for more details.

E:\core-decorators.js>
```

Now If I run tests individually, like

```
mocha --compilers js:babel/register --require babel/polyfill test\unit\autobind.spec.js
```

I get errors like this:
```
  1) @autobind returns a bound instance for a method:
     TypeError: Cannot read property 'equal' of undefined
      at Context.<anonymous> (E:/core-decorators.js/test/unit/autobind.spec.js:48:20)
      at callFn (C:\Users\a\AppData\Roaming\npm\node_modules\mocha\lib\runnable.js:286:21)
      at Test.Runnable.run (C:\Users\a\AppData\Roaming\npm\node_modules\mocha\lib\runnable.js:279:7)
      at Runner.runTest (C:\Users\a\AppData\Roaming\npm\node_modules\mocha\lib\runner.js:421:10)
      at C:\Users\a\AppData\Roaming\npm\node_modules\mocha\lib\runner.js:528:12
      at next (C:\Users\a\AppData\Roaming\npm\node_modules\mocha\lib\runner.js:341:14)
      at C:\Users\a\AppData\Roaming\npm\node_modules\mocha\lib\runner.js:351:7
      at next (C:\Users\a\AppData\Roaming\npm\node_modules\mocha\lib\runner.js:283:14)
      at C:\Users\a\AppData\Roaming\npm\node_modules\mocha\lib\runner.js:314:7
      at done (C:\Users\a\AppData\Roaming\npm\node_modules\mocha\lib\runnable.js:247:5)
      at callFn (C:\Users\a\AppData\Roaming\npm\node_modules\mocha\lib\runnable.js:301:7)
      at Hook.Runnable.run (C:\Users\a\AppData\Roaming\npm\node_modules\mocha\lib\runnable.js:279:7)
      at next (C:\Users\a\AppData\Roaming\npm\node_modules\mocha\lib\runner.js:297:10)
      at Immediate._onImmediate (C:\Users\a\AppData\Roaming\npm\node_modules\mocha\lib\runner.js:319:5)

...
```

Which is also super weird.

So, is there any specific thing I need to do before I can run the tests? 

I'm on Windows 8.1 by the way using node 4.2.1, npm 2.14.7

--- a/package.json
+++ b/package.json
@@ -13,7 +13,7 @@
   "scripts": {
     "build": "babel --out-dir lib src",
     "build-bower": "babel-node scripts/build-bower.js",
-    "test": "npm run build && mocha --compilers js:babel/register --require babel/polyfill 'test/**/*.spec.js'"
+    "test": "npm run build && mocha --compilers js:babel/register --require babel/polyfill \"test/**/*.spec.js\""
   },
   "repository": {
     "type": "git",

--- a/test/exports.spec.js
+++ b/test/exports.spec.js
@@ -12,10 +12,11 @@ const aliases = {
 };
 
 describe('Main package exports', function () {
+  const libPath = path.normalize(`${__dirname}/../lib`);
   let filePaths;
 
   beforeEach(function () {
-    filePaths = glob.sync(`${__dirname}/../lib/*.js`, {
+    filePaths = glob.sync(`${libPath}/*.js`, {
       ignore: ['**/core-decorators.js', '**/*.spec.js']
     });
   });

